package.json

```diff
@@ -10,8 +10,8 @@
     "format": "prettier --write \"./**/src/**/*{ts,js,tsx,jsx,md,mdx}\"",
     "clean": "rimraf node_modules dist",
     "update": "ncu @squidcloud/client @squidcloud/common -u",
-    "publish:public": "npm run build && bash ./publish.sh",
-    "publish:beta": "npm run build && bash ./publish.sh beta"
+    "publish:public": "npm run update && npm run lint && npm run build && bash ./publish.sh",
```

Comment:
I think you need to run `npm install` after running `npm run update` (we could also add that to the `update` command instead).